packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -121,14 +123,15 @@ export async function fetchAndParseAsJSONHelper<T>(
         for (const [key, value] of response.headers.entries()) {
             newHeaders.set(key, value);
         }
-        const res = {
+        res = {
             headers: newHeaders,
             content: JSON.parse(text),
         };
-        return res;
     } catch (e) {
         throwOdspNetworkError(`Error while parsing fetch response: ${e}`, fetchIncorrectResponse, response);
     }
+    assert(res !== undefined, "Response should be defined here");
```

Comment:
Instead of moving return statement around, I think it would be more readable (and less lines of code) to leave as is but add
> assert(false, "unreached");
statement at the end
Even better would be to mark throwOdspNetworkError() as returning never, and that should remove the need to change anything here.

Reply:
done